maas-region/src/helper.py

```diff
@@ -14,6 +17,11 @@
 MAAS_SECRET = Path("/var/snap/maas/common/maas/secret")
 MAAS_ID = Path("/var/snap/maas/common/maas/maas_id")
 MAAS_SERVICE = "pebble"
+MAAS_SSL_CERT_FILEPATH = "/var/snap/maas/common/cert.pem"
```

Comment:
Should we turn those variables to Path rather than strings?

```suggestion
MAAS_SSL_CERT_FILEPATH = Path("/var/snap/maas/common/cert.pem")
```